img/verve.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg width="100%" height="100%" viewBox="0 0 147 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:1.41421;">
+    <g id="Fill-1" transform="matrix(1,0,0,1,0.499974,0.499974)">
+        <path d="M131.991,30.634L131.991,30.655L145.057,30.655L145.057,24.483L120.344,24.483C123.029,28.057 127.23,30.422 131.991,30.634ZM138.879,12.241L117.567,12.241C117.365,13.239 117.258,14.271 117.258,15.328C117.258,16.385 117.365,17.417 117.567,18.414L138.879,18.414L138.879,12.241ZM132.341,0L132.341,0.01C127.436,0.119 123.094,2.511 120.344,6.172L145.057,6.172L145.057,0L132.341,0ZM47.457,30.634L47.457,30.655L60.523,30.655L60.523,24.483L35.81,24.483C38.494,28.057 42.695,30.422 47.457,30.634ZM54.344,12.241L33.033,12.241C32.831,13.239 32.724,14.271 32.724,15.328C32.724,16.385 32.831,17.417 33.033,18.414L54.344,18.414L54.344,12.241ZM47.807,0.01C42.902,0.119 38.56,2.511 35.81,6.172L60.523,6.172L60.523,0L47.807,0L47.807,0.01ZM79.852,0C77.766,0 75.762,0.404 73.896,1.199C72.039,1.992 70.385,3.095 68.98,4.478C67.571,5.863 66.455,7.498 65.661,9.335C64.864,11.178 64.46,13.195 64.46,15.328L64.46,30.655L71.965,30.655L71.965,15.328C71.965,14.242 72.17,13.211 72.575,12.265C72.984,11.312 73.555,10.465 74.272,9.747C74.989,9.03 75.836,8.459 76.789,8.05C77.735,7.645 78.765,7.44 79.852,7.44L83.577,7.44L80.471,0L79.852,0ZM100.775,21.45L92.69,0L84.539,0L97.254,30.655L104.165,30.655L117.011,0L108.795,0L100.775,21.45ZM24.255,0L32.471,0L19.626,30.655L12.715,30.655L0,0L8.15,0L16.236,21.45L24.255,0Z" style="fill:rgb(254,254,254);stroke:black;stroke-width:1px;"/>
```

Comment:
This file was just taken from verve.co, then I added a stroke to the logo so that it was visible against an `#eee` background.